### DISCUSSION TITLE
Elastic Defend Security - Air Gapped server implementation

### DISCUSSION THREAD

**Discussion by udi-mosh** [Security-Onion-Solutions/securityonion#13793] [Ideas] [answered]
Hi Team,

thank you for a great product and for making it easy to maintain even in air gapped networks.
according to this:
https://www.elastic.co/guide/en/security/current/offline-endpoint.html

it would be great if you enable this too as part of the system and update it upon upgrades and adding the ability to update it manually if needed. i am looking at SO as 1 system and i do not want to create another web server just for that.

tnx

**Comment by defensivedepth** [comment, marked as answer]
I have created an issue to track this: https://github.com/Security-Onion-Solutions/securityonion/issues/13829

**Reply by udi-mosh** [reply]
thank you very much
